src/views/search/search.jsx

```diff
@@ -96,6 +95,31 @@ var Search = injectIntl(React.createClass({
         }
         return allTab;
     },
+    getProjectBox: function () {
+        var results = <Grid items={this.state.loaded}
+                            itemType={this.props.tab}
+                            cards={true}
+                            showAvatar={true}
+                            showLoves={false}
+                            showFavorites={false}
+                            showViews={false} />;
+        var searchAction;
```

Comment:
Just to be explicit would you mind setting this to null, i.e. `var searchAction = null;`?